### DISCUSSION TITLE
ArchLinux AUR repository has an old version of GitExtensions

### DISCUSSION THREAD

**Discussion by jasursadikov** [gitextensions/gitextensions#11479] [General]
[This package](https://aur.archlinux.org/packages/gitextensions) is quite old. I've been using GitExtensions back when I was using Windows and was woundering to check if it is available for Linux. 

- I've seen this info in [github.io](https://github.com/gitextensions/gitextensions/wiki/How-To:-run-Git-Extensions-on-Linux) page
- And Linux support was mentioned [here](https://gitextensions.github.io/)

So I was woundering if package would be ever updated. Thank you!

**Comment by gerhardol** [comment]
Yes, there is no update planned right now
Another question today: https://github.com/gitextensions/gitextensions/issues/11475

**Reply by jasursadikov** [reply]
I wounder why then this product claims to have Linux version. It is super outdated...

**Reply by gerhardol** [reply]
There are supposedly still users, Linux is not "marketed" though.
Someone should change https://gitextensions.github.io/ to "was multiplatform".

**Reply by jasursadikov** [reply]
Yeah, I am not sure if archlinux package was submitted by someone from this repo maintainers, but the package is not marked as depricated/obsolete.
https://aur.archlinux.org/packages/gitextensions

**Reply by jasursadikov** [reply]
If it is possible to mark it down as obsolete, that would be nice.
Also when Update prompt comes up and you click on "Update", `.msi` file is being download which is basically installer for Windows.

**Reply by gerhardol** [reply]
I doubt any of the GE maintainers has been involved in Linux distros, if so it was a long time before any current maintainer was involved. Take this with the distro admins.

There could be new 2.x releases if any users supply changes. Then changes like the update check cold be changed (as the update was changed after 2.51.05 was released.
But maintenance has to be done by Linux users. (Is Mono/WinForms still available in distros?)
